source/standards/storing-credentials.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+title: How to store credentials
+last_reviewed_on: 2021-08-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Depending on how you [manage your accounts](/standards/accounts-with-third-parties.html), you, your team and the service you run may have credentials or other secrets that you need to store securely.
+
+## Personal credentials
+
+Examples include login details for:
+
+- GitHub
+- AWS
+- GOV.UK Signon
+
+[Use the password manager built into your browser in the first instance](https://lock.cmpxchg8b.com/passmgrs.html#conclusion). This is simpler than setting up an extra account with a third party and avoids some of the potential issues below.
```

Comment:
Can you elaborate on some of the potential issues referenced here regarding personal passwords for work - I may have missed it but I'm unsure which issues you're referring to? 

I'm also interested in whether we should be advocating folks to use the password manager built into your browser of choice over a third party password manager - I guess this will depend on the potential issues from the previous question.

Reply:
The mechanisms used by third party password managers to integrate with a browser are shaky at best and have a history of being easily tricked by malicious web pages.

I personally advocate using a totally separate password manager with no browser integration as that's akin to building a big obvious bridge from my least trusted environment to my most trusted environment.

I'm sure Ben saved a copy of the original slack thread where we discussed all these things.

Reply:
@karlbaker02 sounds like we might need to rephrase this to make it clearer. The link in the instruction is goes to an article that explains the issue with using third party password managers when combined with a browser extension. We also say this below. How about rephrasing the following sentence to:

> This is [more secure than using a browser extension](https://lock.cmpxchg8b.com/passmgrs.html#conclusion) and you don't need to setup an extra third party account.

> unless synced with some external service, such as their Google account

That's what I do: my passwords are synced with my GDS Google account. Arguably syncing in general is risky if e.g. you're using Safari and syncing your passwords to some random iCloud account that doesn't have 2FA. We don't advocate for or against it. Do you think we need to say something explicitly?

Reply:
I don't think I've read enough on the subject to be honest to say whether I agree that we should be defaulting to use the password managers in-built into browsers over third-party password managers wherever possible for personal work passwords, as well as definitively saying what is more secure (there's more attack surface potentially with an integration, but that doesn't mean the browsers won't do something stupid in the future / introduce a bug which exposes stored credentials etc). @risicle raises a good point about password managers which are completely independent of browser which one might argue are more secure given that there is no integration at all with your browser of choice.

Reply:
That's OK, I'm not an expert on this subject either. A fair few people have contributed to this though, which is reassuring. I agree browsers aren't invulnerable, but that's true of all software.

Specifically about your original comment and my reply: is there anything you'd like me to change?